code/presets/react-webpack/src/loaders/react-docgen-loader.ts

```diff
@@ -69,7 +69,7 @@ const handlers = [...defaultHandlers, actualNameHandler];
 let tsconfigPathsInitialized = false;
 let matchPath: TsconfigPaths.MatchPath | undefined;
 
-export default async function reactDocgenLoader(
+export default function reactDocgenLoader(
```

Comment:
is this exposed to users? i.e. does the presets package export it?

if it does, we will have to leave this as async i think

Reply:
Need input from maintainers for this

Reply:
I checked and it is in the `exports` map, so we should keep this as `async` since it is an exported file

you can just `return Promise.resolve(syncResult)` though